src/server.ts

```diff
@@ -20,26 +20,25 @@ import { token } from 'morgan';
 
 console.time('boot time');
 // Has to import config before any other import uses the configurations
-import { AppConfig, RxNormDbConfig, KafkaConfigurations } from './config';
 import dotenv from 'dotenv';
+import { Server } from 'http';
+import { AppConfig, KafkaConfigurations, RxNormDbConfig } from './config';
+import * as vault from './vault-k8s';
```

Comment:
it would be best to relocate these 2 local imports after the `if`, so that logic doesn't stray away from `dotenv`'s.